packages/dd-trace/src/platform/node/index.js

```diff
@@ -41,7 +37,7 @@ const platform = {
   off: emitter.removeListener.bind(emitter),
   Loader,
   Scope,
-  Exporter
+  Exporter: Exporter()
```

Comment:
Small nit, but this looks like a constructor and could use a better name. An alternative could also be to not return a function from `exporter.js`.

Reply:
I've renamed this to hopefully be clearer